lib/archethic/shared_secrets/node_renewal.ex

```diff
@@ -114,6 +114,35 @@ defmodule ArchEthic.SharedSecrets.NodeRenewal do
     )
   end
 
+  @spec new_origin_shared_secrets_transaction(
+          origin_public_keys :: list(Crypto.key()),
+          secret_key :: binary()
+        ) :: Transaction.t()
+  def new_origin_shared_secrets_transaction(
+        origin_public_keys \\ [<<0::0, 0::0, :crypto.strong_rand_bytes(32)::binary>>],
```

Comment:
Why do you want to define a default origin public keys  ?
 I think you can expect/enforce a list of public keys